modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/struts/FindKBArticleStrutsAction.java

```diff
@@ -80,6 +80,8 @@ public String execute(
 			HttpServletResponse httpServletResponse)
 		throws Exception {
 
+		String backURL = ParamUtil.getString(
+			httpServletRequest, "p_l_back_url");
```

Comment:
Instead of reading the back URL from the parameter like this, why not use the portletDisplay getURLBack method? The parameter name is an implementation detail of the portlet container.

Reply:
I see, it makes sense.